src/test/scala/com/socrata/geospace/lib/client/SodaResponseSpec.scala

```diff
@@ -25,7 +24,7 @@ class SodaResponseSpec extends FunSuiteLike with Matchers {
     val result = Response(200, None)
     val checked = SodaResponse.check(result, 200)
     checked.isFailure should be (true)
-    checked.failed.get should be (JsonParseException)
+    checked.failed.get should be (JsonParseException())
```

Comment:
is this the infamous test you were talking about?

Reply:
Nope.  That was the first nontrivial test over in the RegionCoderServletSepc.